### PR TITLE
Change pk

### PR DESCRIPTION
**Pull request by tyrelsouza**
5 migrations in here, this is what each step encompasses
- Add new field `(primary_key)`
- Populate data (loop through enumerated and set value)
- change `(primary_key)` to being `primary_key=True`
- change `(primary_key)` to be named `(id)`
- double length of `(name)` field and set it to unique

### CODE REVIEW

File: dbfilestorage/migrations/0007_auto_20170206_1940.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```
Comment (tyrelsouza) [inline dbfilestorage/migrations/0007_auto_20170206_1940.py]: Add new field (primary_key)

File: dbfilestorage/migrations/0008_auto_20170206_1945.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```
Comment (tyrelsouza) [inline dbfilestorage/migrations/0008_auto_20170206_1945.py]: Populate data (loop through enumerated and set value)

File: dbfilestorage/migrations/0009_auto_20170206_1954.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```
Comment (tyrelsouza) [inline dbfilestorage/migrations/0009_auto_20170206_1954.py]: change (primary_key) to being primary_key=True

File: dbfilestorage/migrations/0010_auto_20170206_1954.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```
Comment (tyrelsouza) [inline dbfilestorage/migrations/0010_auto_20170206_1954.py]: change (primary_key) to be named (id)

File: dbfilestorage/migrations/0011_auto_20170206_1956.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```
Comment (tyrelsouza) [inline dbfilestorage/migrations/0011_auto_20170206_1956.py]: double length of (name) field and set it to unique

**Comment by PamelaM** [comment]
I would suggest added a test that exercises the name field a bit: Two records, each with a name max-length long, differing only in the last character.

Make sure that the name operations actually work.

File: dbfilestorage/migrations/0012_auto_20170206_2004.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```
Comment (tyrelsouza) [inline dbfilestorage/migrations/0012_auto_20170206_2004.py]: removed the entry `id=models.....` from the models.py file so it's back to the normal django `id` field.

File: dbfilestorage/models.py [hunk 7]
```diff
@@ -7,7 +7,7 @@ class DBFile(models.Model):
     # This is kept as `name` and not something like `md5` because the file
     # operations pass around `name` as the identifier, so it's kept the same
     # to make sense.
```
Comment (morganwahl) [inline dbfilestorage/models.py]: This comment has been stale for a while.

**Comment by morganwahl** [comment]
This could've been done in fewer migrations, but this looks fine.

**Pull request merged**